src/__tests__/04b-lightning-payment.ts

```diff
@@ -303,6 +305,36 @@ functionToTests.forEach(({ fn, name, initialFee }) => {
     const { BTC: finalBalance } = await userWallet1.getBalances()
     expect(finalBalance).toBe(initBalance1)
   }, 60000)
+
+  it(`fails to pay above 2fa threshold without 2fa token`, async () => {
+    // this is needed because this test runs twice
+    if (!userWallet0.user.twoFactor.secret) {
+      const { secret } = userWallet0.generate2fa()
```

Comment:
let's use a `// eslint-disable-next-line  @typescript-eslint/no-non-null-assertion` to remove warning from elint? (also present on github which is nice)